js/plugins/vertexai/src/index.ts

```diff
@@ -146,14 +168,35 @@ export const vertexAI: Plugin<[PluginOptions] | []> = genkitPlugin(
       });
     }
 
+    const embedders = Object.keys(SUPPORTED_EMBEDDER_MODELS).map((name) =>
+      textEmbeddingGeckoEmbedder(name, authClient, { projectId, location })
+    );
+
+    let indexers: IndexerAction<z.ZodTypeAny>[] = [];
+    let retrievers: RetrieverAction<z.ZodTypeAny>[] = [];
+
+    if (options?.vectorSearchIndexOptions) {
+      const defaultEmbedder = embedders[0];
```

Comment:
Should we also let the user specify an embedder (rather than always using a default one)?

Reply:
We should be, though i could have made this more obvious -

https://github.com/firebase/genkit/pull/519/files#diff-3372d5e59486f506715a70db75bb4b22a5605eba23fc43b606906d25c3759a45R74